barbet.json

```diff
@@ -0,0 +1,28 @@
+{
+	"response": [
+		{
+			"maintainer": "Bobby (306bobby)",
+			"oem": "Google",
+			"device": "Pixel 5a",
+			"filename": "crDroidAndroid-14.0-20231231-barbet-v10.0.zip",
+			"download": "https://sourceforge.net/projects/crdroid/files/barbet/10.x/crDroidAndroid-14.0-20231231-barbet-v10.0.zip/download",
+			"timestamp": 1703997608,
+			"md5": "03bc454aa8d8a0ab6455292d0238f260",
+			"sha256": "04377e3677c66fab1ae13c444cab64ef6b0421990786e602f5f04a050565928b",
+			"size": 1236426830,
+			"version": "10.0",
+			"buildtype": "Monthly",
+			"forum": "https://xdaforums.com/t/rom-official-barbet-13-0-14-0-crdroid-9-x-10-x.4646786/",
+			"gapps": "https://sourceforge.net/projects/nikgapps/files/",
```

Comment:
maybe use our gapps?
https://nikgapps.com/crdroid-official

Reply:
I personally don't like Google location services or Google setup so I personally use my own gapps choices. I want that to be a known option for my users as well

Reply:
Agree, yet just saying that if users go for the core gapps, and won't flash other addons, will leave some features as not working
 Reason we don't enforce (and never will) our gapps variant, is to have choice
 I'm just adding some additional context, just so you know
 Anyway, commit is merged, proof we don't enforce specific gapps